### ISSUE TITLE
calendar Quickstart.js should write token using fs.writeFileSync, not fs.writeFile.

### ISSUE THREAD

**Issue by cepm-nate** [google/google-api-nodejs-client#1005]
fs.writeFile does not wait for the writing to complete, and I would end up with an empty /.credentials/calednar-nodejs-quickstart.json file. This of course would crash the app on subsequent starts as it either expects the file to NOT EXIST, or it to exist and have some data.

My fix was to replace `fs.writeFile(TOKEN_PATH, JSON.stringify(token));` with `fs.writeFileSync(TOKEN_PATH, JSON.stringify(token), 'utf8');`, thus insuring that the program actually writes the file.

**Comment by JustinBeckwith** [comment]
Greetings!  Which `quickstart.js` are you referring to?  Is this a tutorial you're following somewhere?

**Comment by cepm-nate** [comment]
Yes, I'm referring to the one at https://developers.google.com/google-apps/calendar/quickstart/nodejs. This is the closest place I can think of to report this issue, as the official node client mentioned.

**Comment by JustinBeckwith** [comment]
Awesome :) Very helpful!

**Comment by JustinBeckwith** [comment]
This tutorial looks to be updated and correct now.  cc @grant

**Comment by cepm-nate** [comment]
The tutorial at https://developers.google.com/google-apps/calendar/quickstart/nodejs still seems to reference fs.writeFile instead of fs.writeFileSync. Was it updated somewhere else?

**Comment by JustinBeckwith** [comment]
Ah!  So yes, the tutorial is out of date.  However, the readFile part looks right.  We're in the process of updating and testing out samples, so apologies for the confusion!
